task1.md

```diff
@@ -93,10 +117,11 @@ Include the role redhat_cop.ee_utilities.ee_builder and have a vars section for
 
 Further documentation for those who are interested to learn more see:
 
-- link1
-- link2
+- [more about include role](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_role_module.html)
```

Comment:
once again doubled up links, may have different focus, but maybe better way of presenting it?

Reply:
ohhh I didn't realize it ended up being the same link, I found an older 2.9 that I liked but then when I went to latest it redirected back to the original link I guess and I didn't catch that